test/common/http/http1/codec_impl_test.cc

```diff
@@ -2251,6 +2251,28 @@ TEST_F(Http1ClientConnectionImplTest, NoContentLengthResponse) {
   EXPECT_TRUE(status.ok());
 }
 
+TEST_F(Http1ClientConnectionImplTest, ResponseWithDuplicatedChunked) {
+  initialize();
+
+  NiceMock<MockResponseDecoder> response_decoder;
+  Http::RequestEncoder& request_encoder = codec_->newStream(response_decoder);
+  TestRequestHeaderMapImpl headers{{":method", "GET"}, {":path", "/"}, {":authority", "host"}};
+  request_encoder.encodeHeaders(headers, true);
+
+  Buffer::OwnedImpl response("HTTP/1.1 200 OK\r\n"
+                             // First chunked header.
+                             "transfer-encoding: chunked\r\n"
```

Comment:
@PiotrSikora for thoughts

I don't *think* this is legal.  Transfer encoding headers are supposed to indicate transformations to the body content

https://tools.ietf.org/html/rfc7230#section-3.3.1

if you send chunked multiple times, you're indicating that the body has been chunked multiple times which is strictly disallowed.  

A sender MUST NOT apply chunked more than once to a message body (i.e., chunking an  already chunked message is not allowed).

Reply:
I agree, I don't think this is allowed.